CHANGELOG.md

```diff
@@ -5,13 +5,32 @@
 ### State Machine Breaking
 
 - [cronos#241](https://github.com/crypto-org-chain/cronos/pull/241) Update ethermint to main and merged statedb refactoring in custom fork.
+- [cronos#289](https://github.com/crypto-org-chain/cronos/pull/289) Update ethermint to `v0.10.0`, with ibc-go reverted to `v2.0.2`, update cosmos-sdk to `v0.45.0-rc1`, the cronos related changes in the dependencies are:
+  - [ethermint#901](https://github.com/tharsis/ethermint/pull/901) support batch evm tx
+  - [ethermint#849](https://github.com/tharsis/ethermint/pull/849) Change EVM hook interface.
+  - [ethermint#809](https://github.com/tharsis/ethermint/pull/809) fix nonce increment issue when contract deployment tx get reverted.
+  - [ethermint#855](https://github.com/tharsis/ethermint/pull/855) unify base fee related logic in the code.
+  - [ethermint#817](https://github.com/tharsis/ethermint/pull/817) Fix eip-1559 logic related to effectiveGasPrice.
+  - [ethermint#822](https://github.com/tharsis/ethermint/pull/822) Update base fee in begin blocker rather than end blocker.
+  - [cosmos-sdk#10833](https://github.com/cosmos/cosmos-sdk/pull/10833) fix reported tx gas used when block gas limit exceeded.
```

Comment:
Not sure we should include cosmos items otherwise we should also do for all other dependency and it would be too long. Maybe just include one that say that we upgrade cosmos version to `v0.45.0-rc1`

Reply:
I basically only included the bug fixes that we are concerned, they are reported on cronos and fixed by us.

Reply:
yeah but we are pretty much concerned by "every bugfix" in any of our dependency right?